### PR TITLE
LA-458 Get Upload Request Groups [skip-ci]

### PR DESCRIPTION
**Pull request by imGok**
uploadrequests

### CODE REVIEW

**Comment by hoangdat** [comment]
Go ahead!!

**Comment by imGok** [comment]
Actived / Closed Text is cropped but I don't know how to fix it :(

File: lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  void initState() {
+    getUploadRequestGroups();
+  }
+
+  void getUploadRequestGroups() {
+    store.dispatch(_getAllUploadRequestGroupsAction([
+      UploadRequestStatus.ARCHIVED,
+      UploadRequestStatus.ENABLED,
+      UploadRequestStatus.CLOSED,
+      UploadRequestStatus.CREATED
```
Comment (hoangdat) [inline lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart]: Why we need to get `UploadRequestGroup` of 4 statuses?

**Review by huynguyennovem — COMMENTED** [review]
/lgtm

**Comment by huynguyennovem** [comment]
> Actived / Closed Text is cropped but I don't know how to fix it :(

`FittedBox` may help you: https://www.youtube.com/watch?v=T4Uehk3_wlY

**Comment by imGok** [comment]
@huynguyennovem @hoangdat fixed this MR

**Comment by huynguyennovem** [comment]
look good!

File: domain/lib/src/model/upload_request_group/upload_request_group_id.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+// more details.
+// You should have received a copy of the GNU Affero General Public License and its
+// applicable Additional Terms for LinShare along with this program. If not, see
+// <http://www.gnu.org/licenses/> for the GNU Affero General Public License version
+//  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
+//  the Additional Terms applicable to LinShare software.
+
+class UploadRequestGroupId {
```
Comment (hoangdat) [inline domain/lib/src/model/upload_request_group/upload_request_group_id.dart:32]: Should add `with EquatableMixin`

File: testshared/pubspec.yaml [hunk 3]
```diff
@@ -3,6 +3,7 @@ description: LinShare testshared package.
 version: 0.0.1
 author:
 homepage: 'https://www.linshare.org'
+publish_to: none
```
Comment (hoangdat) [inline testshared/pubspec.yaml:6]: why we need to add this?

File: data/pubspec.yaml [hunk 4]
```diff
@@ -3,6 +3,7 @@ description: LinShare data package.
 version: 0.0.1
 author:
 homepage: 'https://www.linshare.org'
+publish_to: none
```
Comment (hoangdat) [inline data/pubspec.yaml:6]: same

**Review by hoangdat — APPROVED** [review]

**Pull request merged**